### PR TITLE
Units_Generator - now generates planetary densities

### PR DESCRIPTION
**Pull request by oriash93**
related to #63

### CODE REVIEW

File: units_cpp/src/units/headers/units/constants/planetary_densities.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+#include <units/density_units.h>
+
+
+namespace units {
+namespace planetary_densities
+{
+
+//the ratio of the mass of the sun to the volume of space it takes up
+constexpr density::KilogramsPerLiter sunDensity{1.4};
```
Comment (saroad2) [inline units_cpp/src/units/headers/units/constants/planetary_densities.h:16]: are you sure it's kilograms per liter? that sounds odd...

Comment (saroad2) [inline units_cpp/src/units/headers/units/constants/planetary_densities.h:16]: are you sure it's kilograms per liter? that sounds odd...

File: units_generator/src/main/resources/data/units_constants.json [hunk 2]
```diff
@@ -184,7 +243,7 @@
 						"denominators" : ["seconds", "seconds"]
 					},
 					"value" : 6.67408e-11,
-					"description" : "a constant reguarding the force between to objects, depending on their masses and distance"
+					"description" : "a constant regarding the force between to objects, depending on their masses and distance"
```
Comment (saroad2) [inline units_generator/src/main/resources/data/units_constants.json]: "two objects", not "to objects". my bad.

**Review by saroad2 — CHANGES_REQUESTED** [review]

File: units_generator/src/main/resources/data/units_constants.json (hunk 2 above)
Comment (oriash93) [reply units_generator/src/main/resources/data/units_constants.json]: will fix it

File: units_cpp/src/units/headers/units/constants/planetary_densities.h (hunk 1 above)
Comment (oriash93) [reply units_cpp/src/units/headers/units/constants/planetary_densities.h:16]: originally it's gram per cubic centimeter, which is equal to kilogram per liter.
I think it's okay to leave it this way

Comment (oriash93) [reply units_cpp/src/units/headers/units/constants/planetary_densities.h:16]: originally it's gram per cubic centimeter, which is equal to kilogram per liter.
I think it's okay to leave it this way

Comment (saroad2) [reply units_cpp/src/units/headers/units/constants/planetary_densities.h:16]: what about changing the name of the unit scale itself to "grams per centimeter"? what is more in common use?

Comment (oriash93) [reply units_cpp/src/units/headers/units/constants/planetary_densities.h:16]: I think it's used the same. besides, we don't have "cubic centimeters" scale in volume so it's easier 😛

Comment (saroad2) [reply units_cpp/src/units/headers/units/constants/planetary_densities.h:16]: ok

**Pull request merged**